_posts/2024-04-16-mai-win-sr2024.md

```diff
@@ -0,0 +1,176 @@
+
+## The game: Final Frontier
+
+{% include figure.html
+           class="right"
+           src="/images/content/blog/sr2024/arena.png"
+           caption="The 'Final Frontier' arena"
+           alt="
+            A square arena with each of four 'Planet' scoring zones placed along the edges.
+            Those surround an 'Asteroid field' featuring 16 'Asteroids' and in the very center is an 'Egg' on a raised 'Nest'.
+            Robots start in the middle of their home Planet, with a 'Spaceship' (also on their Planet) to their right.
+          "
+%}
+
+This year’s game, *Final Frontier*, plunged teams into an interstellar battle for dominance among four formidable civilizations, all vying for control of the cosmos. Teams had to retrieve Asteroids scattered throughout the Asteroid field and transport them back to their home Planet or, even better, their intergalactic Spaceship currently under construction.
+
+However the stakes are high as within the confines of the asteroid belt lies The Egg -- a cosmic time bomb that, if disturbed, would spell chaos. Teams had to strategise, navigate the treacherous field, and decide whether to safeguard their own civilization or sabotage their rivals. With Asteroids worth points, Spaceships to manage, and The Egg lurking, the universe had never been this competitive.
+
+## Challenges
+
+Following on from last year’s success, this year's competition offered teams a set of Challenges to be completed throughout the competition year. These encouraged teams to start work on their robots early and covered movement, mechanics, and vision. They could be approached in any order, and completing challenges before certain deadlines earned the teams bonus league points.
+
+## Virtual Competition
+
+The first part of our league this year was a virtual competition, broadcast live in March. Across twenty-one matches, teams competed in a simulated version of the main competition, testing their coding skills against one another. Wisbech Grammar School offered the best performance on the day, coming 1st in all four of their matches, though the team from Haberdashers’ Schools remained at the top of the leaderboard due to points they'd earned from the Challenges.
+
+## League Matches
+
+During the in person event teams participated in a further 69 league matches, allowing the robots to improve and evolve both their construction and strategies. The robots displayed impressive agility, precision and ingenuity as they navigated the arena to capture Asteroids (and Spaceships in some cases) as well as outmanoeuvre their opponents.
+
+At the end of the league there were only two league points separating the top three teams (The Ladies' College Guernsey, Wisbech Grammar School and [Royal Grammar School Guildford][RGS]), with the next few teams not far behind.
```

Comment:
```suggestion
At the end of the league there were only two league points separating the top three teams ([The Ladies' College Guernsey][TLC], [Wisbech Grammar School][WGS] and [Royal Grammar School Guildford][RGS]), with the next few teams not far behind.
```

Reply:
I'd been following the wikipedia rule of not linking things already linked to.

Reply:
ec4778f